### DISCUSSION TITLE
Similar alternatives for KDE Plasma?

### DISCUSSION THREAD

**Discussion by Castlemark** [paperwm/PaperWM#746] [Q&A]
This is just a question, I tried this extension on a friend's computer and loved it.

The thing is, I use KDE Neon and therefore can't use the extension. Does anyone know of anything similar for Plasma?

**Comment by hedning** [comment]
Not that I'm aware of unfortunately.

**Comment by Castlemark** [comment]
That's a shame. Maybe I'll try to replicate it with kwin scripts when i have a free weekend.

**Comment by KiaraGrouwstra** [comment]
@Castlemark if you're opening a repo for this, please link it here. :)

**Comment by perin-lionshead** [comment]
@Castlemark  I would also love a KDE version, so if you have any sort of lead, or inkling, or need help with something, let me know!

**Comment by hedning** [comment]
Not KDE, but there's apparently some work on a stand-alone wayland window-manager inspired by paperwm here: https://gitlab.com/cardboardwm/cardboard

**Comment by KiaraGrouwstra** [comment]
curious how people feel about the UX of that one compared to this :)

**Comment by Thesola10** [comment]
You can install GNOME Shell on KDE Neon, after all it's just a Linux distro like any other.

**Comment by Polisson** [comment]
Recently appeard something:
[Karousel](https://github.com/peterfajdiga/karousel) kwin script. Available also in [store](https://www.pling.com/p/2045724).
